pyscriptjs/src/plugin.ts

```diff
@@ -60,6 +60,26 @@ export class Plugin {
      */
     afterPyScriptExec(options: {interpreter: Interpreter, src: string, pyScriptTag: PyScriptTag, result: any}) {}
 
+    /** The source of the <py-repl> tag has been fetched and its output-element determined;
+     * we're about to evaluate the source using the provided runtime
+     *
+     * @param options.interpreter The Runtime object that will be used to evaluated the Python source code
```

Comment:
Do you think runtime here is okay or should we call it interpreter object? 🤔 

```suggestion
     * @param options.interpreter The Interpreter object that will be used to evaluate the Python source code
```